turbopack/crates/turbopack-ecmascript/src/lib.rs

```diff
@@ -407,6 +407,36 @@ impl EcmascriptAnalyzable for EcmascriptModuleAsset {
     }
 }
 
+#[turbo_tasks::function]
+async fn determine_module_type_for_directory(
+    context_path: Vc<FileSystemPath>,
+) -> Result<Vc<ModuleTypeResult>> {
+    let find_package_json =
+        find_context_file(context_path, package_json().resolve().await?).await?;
+    let FindContextFileResult::Found(package_json, _) = *find_package_json else {
+        return Ok(ModuleTypeResult::new(SpecifiedModuleType::Automatic));
+    };
+
+    // analysis.add_reference(PackageJsonReference::new(package_json));
```

Comment:
What about this?

Reply:
might be relevant for nft... 
Not sure why it's commented out...

Reply:
But I only moved the code. It was commented out before